packages/experimental-app-router/src/server/routeHandler/tokenHandler.ts

```diff
@@ -80,7 +80,7 @@ export async function tokenHandler(req: Request) {
      * and expiration.
      */
 
-    const res = NextResponse.json(data, {
+    const res = new NextResponse(JSON.stringify(data), {
```

Comment:
Was experiencing some weird testing behavior with `NextResponse.json`, it seems others are too:

https://github.com/vercel/next.js/issues/48524

It appears it's related to earlier Node versions, so instead of calling the `json` method, I instead opted to just stringify the JSON myself.